.github/workflows/python-package.yml

```diff
@@ -131,6 +131,36 @@ jobs:
         name: py-opentimelineio-codecov
         fail_ci_if_error: true
 
+  py_build_test_mingw64:
```

Comment:
I'm wondering if this could just be integrated into the `py_build_test` matrix using an [include](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations) with an `if` on the msys2 action. Would that work instead of creating a new job?

Reply:
I started doing it that way, but it got really messy.  This was much simpler to get working.  I can take another crack at it if that would be preferred.

I'm also not uploading the artifacts as they are not supported by pypi.